services/storage/in_memory_storage.go

```diff
@@ -358,12 +359,97 @@ func (storage *InMemoryStorage) resetCommitStorage(blockHeight primitives.BlockH
 	storage.commitStorage[blockHeight] = views
 	return views
 }
+
 func (storage *InMemoryStorage) resetViewChangeStorage(blockHeight primitives.BlockHeight) map[primitives.View]map[MemberIdStr]*interfaces.ViewChangeMessage {
 	views := make(map[primitives.View]map[MemberIdStr]*interfaces.ViewChangeMessage)
 	storage.viewChangeStorage[blockHeight] = views
 	return views
 }
 
+func (storage *InMemoryStorage) GetMessagesLogs(blockHeight primitives.BlockHeight, view primitives.View) []interfaces.MemberMessagesLog {
```

Comment:
if you have the timestamp each message was received, or at least preserve the order in which they came in, that could be a nice addition to the data we output to the logs.

perhaps add a //TODO to add this?

Reply:
I'm not sure it adds any reliable information regarding other nodes that we can rely on - kinda mixing network and in-node processing timing issues, but I added a note to reconsider